_posts/en/2017-03-20-the-polite-code-reviewer.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: default
+title:  "The Polite Code Reviewer"
+date:   2017-03-20 17:50:00
+categories:
+  - en
+  - working-with-others
+---
+
+## The Polite Code Reviewer
+
+### TL;DR
+
+ - Code Review is an opportunity educate,
```

Comment:
maybe?
`Code review is an opportunity to educate, and learn at the same rate`
I think something around this would be better. 🌱